superset-frontend/src/components/ErrorMessage/ErrorAlert.tsx

```diff
@@ -163,11 +169,17 @@ export default function ErrorAlert({
           onHide={() => setIsModalOpen(false)}
           title={
             <div className="header">
-              <Icon
-                className="icon"
-                name={level === 'error' ? 'error-solid' : 'warning-solid'}
-                color={supersetTheme.colors[level].base}
-              />
+              {level === 'error' ? (
```

Comment:
If this check comes up a lot, it might be worth making a new component that renders the correct type of icon, so  that you can just do `<AlertIcon level={level} />`

Reply:
Yah i make a note of this. But I think these were the only two with those conditions.